Havit/Havit/Screens/CategoryContents/View/CategoryContentsViewController.swift

```diff
@@ -6,24 +6,184 @@
 //
 
 import UIKit
+import SnapKit
+import RxSwift
 
-class CategoryContentsViewController: UIViewController {
+class CategoryContentsViewController: BaseViewController {
+    var searchController: UISearchController!
+    var mainView: UIView!
+    var filterView: UIView!
+    var totalLabel: UILabel!
+    var changeShowButton: UIButton!
+    var sortButton: UIButton!
+    
+    private var filterCollectionView: UICollectionView!
+    private var contentsCollectionView: UICollectionView!
 
     override func viewDidLoad() {
         super.viewDidLoad()
-
-        // Do any additional setup after loading the view.
+        setUI()
+        setAutoLayouts()
+        
     }
     
+    func setUI() {
+        // TODO: 네비게이션바 생성 (메인화면에서 Coordinator로 진입)
+        
+        // 검색바 생성
+        let searchController = UISearchController(searchResultsController: nil)
+        searchController.searchBar.placeholder = "원하는 콘텐츠를 검색하세요."
+        self.navigationItem.searchController = searchController
+        self.navigationItem.title = "카테고리"
+        searchController.hidesNavigationBarDuringPresentation = false
```

Comment:
p3;
searchController 를 설정하는 곳은 한곳에 모아서 작성해주시는게 가독성 측면에서 좋을 것 같습니다
```suggestion
        let searchController = UISearchController(searchResultsController: nil)
        searchController.searchBar.placeholder = "원하는 콘텐츠를 검색하세요."
        searchController.hidesNavigationBarDuringPresentation = false
        self.navigationItem.searchController = searchController
        self.navigationItem.title = "카테고리"
    
```

Reply:
넵 함수로 따로 빼도록 하겠습니다!